framework/interface/ui_handlers.py

```diff
@@ -440,11 +441,12 @@ class FileRedirectHandler(custom_handlers.UIRequestHandler):
     SUPPORTED_METHODS = ('GET')
 
     def get(self, file_url):
+	file_url = file_url[1:]
```

Comment:
It doesn't show any indentation error in my kali and also doesn't show any false indentation in my develop branch files. I don't why it is showing in the commit.